documentation/docs/guides/svg-react-components.md

```diff
@@ -0,0 +1,29 @@
+# SVG React Components
+
+React components can be generated from SVG files using [https://github.com/smooth-code/svgr](svgr).
+
+Suppose you have the following icons:
+
+```
+src/
+└── icons
+    ├── foo.svg
+    └── bar.svg
+```
+
+Running `npx @svgr/cli -d src/icons src/icons` will generate the corresponding React components:
```

Comment:
Can we add what you would want to change in your npm scripts for this to work using fusion dev command?

Reply:
I think for svg images, running this once and committing the JS would probably be best. I'm not sure having a long-running process for this would be good.